lib/emittr/events.rb

```diff
@@ -39,18 +39,37 @@ def once(event, &block)
         on(event, &off_block)
       end
 
+      def on_any(&block)
+        on(:any, &block)
+      end
+
+      def off_any(&block)
+        off(:any, &block)
+      end
+
+      def once_any(&block)
+        once(:any, &block)
+      end
+
       def emit(event, *payload)
         return unless listeners.key? event
```

Comment:
the can call `#emit` without the event, but I think we're not handling this case properly. the best approach for this is to throw an argument error. what do you think?